sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -88,12 +90,25 @@ private[spark] object HiveUtils extends Logging {
       |   <code>${builtinHiveVersion}</code> or not defined.
       | 2. "maven"
       |   Use Hive jars of specified version downloaded from Maven repositories.
-      | 3. A classpath in the standard format for both Hive and Hadoop.
+      | 3. "path"
+      |   Use Hive jars configured by `spark.sql.hive.metastore.jars.path` in comma separated format
+      |   support both local or remote paths, it should always be fully qualified URL to indicate
+      |   other file systems.
+      | 4. A classpath in the standard format for both Hive and Hadoop.
```

Comment:
`This property can be one of three options` should be updated to use `four options`

Reply:
Done